x/auth/types/expected_keepers.go

```diff
@@ -19,4 +19,9 @@ type BankKeeper interface {
 type AccountsModKeeper interface {
 	SendModuleMessageUntyped(ctx context.Context, sender []byte, msg protoiface.MessageV1) (protoiface.MessageV1, error)
 	IsAccountsModuleAccount(ctx context.Context, accountAddr []byte) bool
+	NextAccountNumber(ctx context.Context) (accNum uint64, err error)
+
+	// InitAccountNumberSeq is use to set accounts module account number with value
+	// of auth module current account number
+	InitAccountNumberSeq(ctx context.Context, currentAccNum uint64) error
```

Comment:
should we name this Unsafe? its a pretty dangerous thing to have public

Reply:
good idea

Reply:
done